public/components/Tag/Display.react.js

```diff
@@ -64,6 +64,41 @@ class TagDisplay extends React.Component {
       return !validateTag(this.props.tag).length;
     }
 
+    renderDeleteButton() {
+      if (this.props.config.permissions["tag_super_admin"]) {
+        return <ConfirmButton className="tag__delete" onClick={this.deleteTag.bind(this)} buttonText="Delete Tag" />
+      } else {
+        return (
+          <div>
+            <ConfirmButton className="tag__delete tag__delete--disabled" disabled={true} buttonText="Delete Tag" />
+            <span className="tag-edit__label">You do not have permission to delete this tag.</span>
+          </div>
+        )
+      }
+    }
+
+    renderSaveBanner() {
+      var permitted = this.props.config.permittedTagTypes;
+
+      if (permitted.some((e, i, a) => e == this.props.tag.type)) {
+        return <SaveButton isHidden={!this.isTagDirty() || !this.isTagValid()}
+                           onSaveClick={this.saveTag.bind(this)}
+                           onResetClick={this.resetTag.bind(this)}/>
+      }
+    }
+
+    renderWarningBar() {
+      if (!this.props.warning) {
```

Comment:
I think this should be specific to tagEditable    `if (!this.props.tagEditable)` not show the system wide warning